src/components/logo/logo.tsx

```diff
@@ -133,9 +134,19 @@ class Logo extends Component<any, any> {
           >
           <img className={style.icon} src={props.config.img} />
         </a>
+        ) : (
+          <span
+            className={style.controlButton}
+            aria-label={props.config.text || props.logoText}
+            tabIndex={0}
```

Comment:
why is there a tab index if the element is not clickable ?

Reply:
You are right, the element doesn't need to be navigable any more